.github/ISSUE_TEMPLATE/sensor_request.md

```diff
@@ -9,98 +9,36 @@ assignees: '@teresaho-biosensics'
-- [ ] Server ECO Section I approved
-- [ ] Update Server `CHANGELOG.md` "Unreleased" section
-- [ ] Confirm any updated Aptible test server configuration values
-- [ ] Server link to source code:
-
-Application:
-- [ ] Update Application `CHANGELOG.md` "Unreleased" section
-- [ ] Create application release issue in application repository using the Release Issue template
-- [ ] Link this issue to the correct release version
-- [ ] Request App ECO number from [Quality](mailto:quality@biosensics.com)
-- [ ] App ECO Section I submitted to [Quality](mailto:quality@biosensics.com)
-      (`G:\Shared drives\QSR\BS-02 Quality System Procedures (SOPs)\BS-02-0021_Software Changes, Releases and Documentation`)
-- [ ] App ECO Section I approved
-- [ ] CLI tools set up
-- [ ] Confirm internal telehealth meeting room
-- [ ] Set GitHub secrets per server configuration values and telehealth meeting room
-- [ ] APK connects to server (indicate if Test, Production, UAT or QA)
-- [ ] Send Application RC build to operations team for V&V
-- [ ] App link to source code:
-
-**@biosensics/operations & @biosensics/quality :**
-- [ ] V&V Document Released
-- [ ] Complete DCO and submit to [Quality](mailto:quality@biosensics.com)
-- [ ] Hexnode portal/policy set up
-- [ ] Mobile device operating system is correct
-- [ ] Mobile device model is correct (version, iOS/Android, etc.)
-- [ ] Sensor firmware is correct
-- [ ] Device enrolled in Hexnode
-- [ ] Equipment set up for V&V execution
-- [ ] Parse account created in correct server
-- [ ] List parse account username:
-
-*Ensure all above items have been completed before continuing*
+***Pre-Manufacturing Checklist:***
 
 **@biosensics/test-and-manufacturing :**
-- [ ] V&V executed
-- [ ] Completed V&V submitted to [Quality](mailto:quality@biosensics.com)
-
-**@biosensics/quality :**
-- [ ] Approve completed V&V
+- [ ] Manufacturing Report Created for the Month
+- [ ] Welder Maintenence Completed for the Month [Date Completed: ]
+- [ ] Welder Logs for Previous Month Uploaded [Link: ]
+- [ ] Batch Number Updated in Spreadsheet [Updated Batch Number: ]
 
-**@biosensics/engineering :**
 
-Server:
-- [ ] Send tagged release source code zip to @biosensics/quality
-- [ ] Create server release
-- [ ] Deploy server release to Aptible production server
-- [ ] Confirm any updated Aptible production server configuration values
-- [ ] Merge `release` branch into `develop` on server repository
-
-Application:
-- [ ] Merge `release` branch into `master` on Application repository
-- [ ] Send release APK and installer to operations and quality for production use
+***Manufacturing Checklist:***
 
+**@biosensics/test-and-manufacturing :**
 
-**@biosensics/quality :**
-- [ ] App ECO closed & filed to QSR
-- [ ] Server ECO closed & filed to QSR
-- [ ] Insert file path(s) to documentation (ECO, V&V etc):
+|   	| Board ID 	| Manu Tested 	| Firmware Uploaded 	| Welded 	| Calibrated 	| Epoxied 	| Verified and Ready for Shipment 	|
```

Comment:
Do you want to capture a YES/NO or specific information? E.g., `Board ID, Mac Address, Manufacture Test Complete, Firmware Version, Weld Number, Calibrated, Epoxied, Verified & Ready for Shipment`